### ISSUE TITLE
Firefox tab cursor gets stuck on grid examples page

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#5357]
[Forum post](https://www.bryntum.com/forum/viewtopic.php?f=51&t=22527&p=111359#p111359)

"Tabbing behaviour is inconsistent between Chrome, Firefox and Safari. While I imagine the majority of this is browser specific could I have clarification on an issue within Firefox. The image attached shows the tabbing order and as soon as the tab cursor enters the grid you are no longer able to continue tabbing through elements. From what I can tell safari uses the same tab order but does not get stuck instead going though the order indicated on the screenshot. With chrome having very different functionality.

![Bryntum Examples tabbing order screenshot.png](https://www.bryntum.com/forum/download/file.php?id=19023)

To reproduce on a specific example, the "Tree with locked columns" grid when tabbed into on firefox will highlight the scrollbar but you will be unable to continue tabbing though other elements."

I also attached a video for more clarification

https://user-images.githubusercontent.com/16693227/193914897-351e4e0e-5437-44f7-929f-333e5584a407.mp4
